fuzz/fuzz_targets/fuzz_common.rs

```diff
@@ -12,6 +12,7 @@ use std::io::{Seek, SeekFrom, Write};
 use std::os::fd::{AsRawFd, RawFd};
 use std::process::{Command, Stdio};
 use std::sync::atomic::Ordering;
+use std::sync::mpsc;
```

Comment:
I guess this is a leftover and can be removed? Or did you forget to commit some changes?
```suggestion
```